docs/DeveloperGuide.md

```diff
@@ -1036,6 +1036,78 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes from step 2.
 
+
+**Use case: UC05 - Assign task a student**
+
+**MSS**
+1. User requests TAPA to assign a task to a student, by specifying their studentId and task name or assign tasks to all the student taking a module, by specifying module code and task name.
```

Comment:
```suggestion
1. User requests TAPA to assign a task to a student. (This can be done for an individual student or for all students taking the same module)
```

If i remember correctly, use cases should not have too much implementation details so just a suggestion to remove some details.